corehq/apps/reports/tasks.py

```diff
@@ -337,11 +338,33 @@ def extract_form_info(form, properties=None, case_ids=case_ids):
 
         return form_info
 
-    key = [domain, app_id, xmlns]
-    form_ids = {f['id'] for f in XFormInstance.get_db().view("attachments/attachments",
-                                                             start_key=key + [startdate],
-                                                             end_key=key + [enddate, {}],
-                                                             reduce=False)}
+    if not export_is_legacy:
+        query = (FormES()
+                 .domain(domain)
+                 .app(app_id)
+                 .xmlns(xmlns)
+                 .submitted(gte=parse(startdate), lte=parse(enddate))
+                 .remove_default_filter("has_user"))
+        form_ids = set()
+        for form in query.scroll():
+            try:
+                for attachment in form['_attachments'].values():
```

Comment:
also, somewhat related, are we going to end up with a state where X number of forms in ES have `_attachments` and Y number have `external_blobs` unless we did a full reindex? easy solution could be to have the transform function move `external_blobs` to `_attachments`. realize that's kind of confusing though